### ISSUE TITLE
500 errors when trying to get JSON of per-DJ albums or artists chart

### ISSUE THREAD

**Issue by matthazinski** [wuvt/wuvt-site#187]
Albums:
```
[matt@teselecta ~]$ wget -S -q --header "Accept: application/json" -O - https://www.wuvt.vt.edu/playlists/charts/albums/dj/77
  HTTP/1.1 500 INTERNAL SERVER ERROR
  Server: nginx
  Date: Tue, 01 Dec 2015 04:54:25 GMT
  Content-Type: text/html
  Content-Length: 291
  Connection: keep-alive
```

```
Message type:       ERROR
Time:               2015-11-30 23:54:25,144

Exception on /playlists/charts/albums/dj/77 [GET]
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./wuvt/trackman/views.py", line 274, in charts_albums_dj
    'results': [(x[0].serialize(), x[1]) for x in results],
AttributeError: 'unicode' object has no attribute 'serialize'
```

Artists:
```
[matt@teselecta ~]$ wget -S -q --header "Accept: application/json" -O - https://www.wuvt.vt.edu/playlists/charts/artists/dj/77
  HTTP/1.1 500 INTERNAL SERVER ERROR
  Server: nginx
  Date: Tue, 01 Dec 2015 04:56:38 GMT
  Content-Type: text/html
  Content-Length: 291
  Connection: keep-alive
```
```
Message type:       ERROR
Time:               2015-11-30 23:56:38,454

Exception on /playlists/charts/artists/dj/77 [GET]
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./wuvt/trackman/views.py", line 312, in charts_artists_dj
    'results': [(x[0].serialize(), x[1]) for x in results],
AttributeError: 'unicode' object has no attribute 'serialize'
```

Interestingly enough, getting tracks JSON is fine.

**Comment by matthazinski** [comment]
Also broken:
* https://www.wuvt.vt.edu/playlists/charts/artists
* https://www.wuvt.vt.edu/playlists/charts/albums

**Comment by mutantmonkey** [comment]
Fixed in b694301f15122feabe19aa93c2587025631b3f9d
